kfs-purap/src/main/java/org/kuali/kfs/module/purap/service/impl/PdpExtractServiceImpl.java

```diff
@@ -549,7 +549,17 @@ protected PaymentDetail populatePaymentDetail(VendorCreditMemoDocument creditMem
 
         paymentDetail.setInvoiceDate(creditMemoDocument.getCreditMemoDate());
         paymentDetail.setOrigInvoiceAmount(creditMemoDocument.getCreditMemoAmount().negated());
-        paymentDetail.setNetPaymentAmount(creditMemoDocument.getFinancialSystemDocumentHeader().getFinancialDocumentTotalAmount().negated());
+        /* UAF-4518 : rationale not extend this class for this mod
```

Comment:
The justification seems right not to extend this class, however, please remove the commentary from the code. The explanation should be either in the Jira ticket or in the Git Pull request comment, but not in the code.